app/views/ctc/portal/portal/home.html.erb

```diff
@@ -69,9 +69,6 @@
       <div class="spacing-above-35">
         <%= link_to t('views.ctc.portal.home.contact_us'), new_ctc_portal_message_path, class: "button button--full-width" %>
       </div>
-      <% if @can_submit_documents %>
```

Comment:
removing this because we only added this last season to facilitate the upload of identity documents.

Reply:
im actually going to check with product to see if it should be removed or not